lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
I don't think it's practical or wise to require error messages to be out here. There's a significant chance of forgetting to keep these in sync with actual code.